system-addon/content-src/styles/_theme.scss

```diff
@@ -0,0 +1,80 @@
+// Default theme
+.outer-wrapper {
+  --newtab-background-color: $grey-10;
+  --newtab-text-primary-color: $grey-90;
+  --newtab-text-secondary-color: $grey-50;
+  --newtab-text-conditional-color: $grey-60;
+  --newtab-link-primary-color: $blue-60;
+  --newtab-link-secondary-color: $teal-70;
+  --newtab-border-primary-color: $grey-40;
+  --newtab-border-secondary-color: $grey-30;
+  --newtab-icon-primary-color: $grey-90-80;
+  --newtab-icon-secondary-color: $grey-90-60;
+  --newtab-icon-tertiary-color: $grey-30;
+  --newtab-textbox-border: $grey-90-20;
+  --newtab-textbox-color: inherit;
+  --newtab-active-element-color: $grey-20-60;
+  --newtab-button-primary-color: $blue-60;
+  --newtab-button-secondary-color: inherit;
+  --newtab-contextmenu-button-color: $white;
+  --newtab-modal-color: $white;
+  --newtab-overlay-color: $grey-20;
+  --newtab-searchbox-background-color: $white;
+  --newtab-searchbox-border-color: transparent;
+  --newtab-searchbox-text-color: inherit;
+  --newtab-searchbox-icon-color: $grey-90-40;
+  --newtab-section-header-text-color: $grey-50;
+  --newtab-section-navigation-text-color: $grey-50;
+  --newtab-section-navigation-link-color: $blue-40;
+  --newtab-section-active-contextmenu-color: $grey-90;
+  --newtab-topsites-label-color: inherit;
+  --newtab-card-background-color: $white;
+  --newtab-inner-box-shadow-color: $black-10;
+}
+
+// Dark theme
+.outer-wrapper.dark-theme {
+  --newtab-background-color: $grey-80;
+  --newtab-text-primary-color: $grey-10;
+  --newtab-text-secondary-color: $grey-10-40;
+  --newtab-text-conditional-color: $grey-10;
+  --newtab-link-primary-color: $blue-40;
+  --newtab-link-secondary-color: $blue-40;
+  --newtab-border-primary-color: $grey-10-80;
+  --newtab-border-secondary-color: $grey-10-40;
+  --newtab-icon-primary-color: $grey-10-80;
+  --newtab-icon-secondary-color: $grey-10-40;
+  --newtab-icon-tertiary-color: $grey-10-20;
+  --newtab-textbox-border: $grey-10-40;
+  --newtab-textbox-color: $grey-80;
+  --newtab-active-element-color: $grey-60;
+  --newtab-button-primary-color: $blue-60;
+  --newtab-button-secondary-color: $grey-70;
+  --newtab-contextmenu-button-color: $grey-80;
+  --newtab-modal-color: $grey-90;
+  --newtab-overlay-color: $grey-80;
+  --newtab-searchbox-background-color: $grey-60;
+  --newtab-searchbox-border-color: $grey-10-40;
+  --newtab-searchbox-text-color: $grey-10-60;
+  --newtab-searchbox-icon-color: $grey-10-60;
+  --newtab-section-header-text-color: $grey-10-80;
+  --newtab-section-navigation-text-color: $grey-10-80;
+  --newtab-section-active-contextmenu-color: $white;
+  --newtab-topsites-label-color: $grey-10-80;
+  --newtab-card-background-color: $grey-90;
+  --newtab-inner-box-shadow-color: $grey-60;
+}
+
+// scss variables related to the theme.
```

Comment:
I moved these variables below into this file because some depend on the CSS variables defined above. But I guess they can go back to _variables.scss even though that's imported first because the vars don't get evaluated until runtime anyway.